tools/nuget/README.md

```diff
@@ -11,11 +11,11 @@ All build scripts follow the same routine:
 
 
 * `build.win.ps1`
-    * A [PowerShell](https://github.com/PowerShell/PowerShell) script to build Windows native libraries. By default, both `win-x86` and `win-x64` are built. To build only one specific Runtime Identifier, pass it to the script as a parameter (`./build.win.ps1 win-x64`)  
```

Comment:
Two trailing whitespaces mean a linebreak in markdonw.

Reply:
Related discussion: https://github.com/SFML/CSFML/pull/218#discussion_r1433287784

Reply:
Yep

Before:

![image](https://github.com/SFML/CSFML/assets/920861/ef07aebd-7aa1-47c2-9474-c7fcab05777a)

After:

![image](https://github.com/SFML/CSFML/assets/920861/6b68563c-b313-4699-9dba-d4b4fa1de6b7)

It's not really bad here, but I think kimci86 mentioned that on another PR once before. Something to keep in mind when "formatting" markdown files.